aspnetcore/fundamentals/index.md

```diff
@@ -1,216 +1,206 @@
 
-ASP.NET Core includes a rich set of built-in middleware:
+By convention, a middleware component called "XYZ" is added to the pipeline by invoking a `UseXYZ` extension method in the `Configure` method.
 
-* [Static files](xref:fundamentals/static-files)
-* [Routing](xref:fundamentals/routing)
-* [Authentication](xref:security/authentication/index)
-* [Response Compression Middleware](xref:performance/response-compression)
-* [URL Rewriting Middleware](xref:fundamentals/url-rewriting)
+ASP.NET Core includes a rich set of built-in middleware, and you can write your own custom middleware. [Open Web Interface for .NET (OWIN)](xref:fundamentals/owin), which allows web apps to be decoupled from web servers, is supported in ASP.NET Core apps.
 
-[OWIN](http://owin.org)-based middleware is available for ASP.NET Core apps, and you can write your own custom middleware.
-
-For more information, see [Middleware](xref:fundamentals/middleware/index) and [Open Web Interface for .NET (OWIN)](xref:fundamentals/owin).
+For more information, see <xref:fundamentals/middleware/index> and <xref:fundamentals/owin>.
 
 ::: moniker range=">= aspnetcore-2.1"
 
 ## Initiate HTTP requests
 
-For information about using `IHttpClientFactory` to access `HttpClient` instances to make HTTP requests, see [Initiate HTTP requests](xref:fundamentals/http-requests).
+<xref:System.Net.Http.IHttpClientFactory> is available to access <xref:System.Net.Http.HttpClient> instances to make HTTP requests.
+
+For more information, see <xref:fundamentals/http-requests>.
 
 ::: moniker-end
 
 ## Environments
 
-Environments, such as "Development" and "Production", are a first-class notion in ASP.NET Core and can be set using environment variables.
+Environments, such as *Development* and *Production*, are a first-class notion in ASP.NET Core and can be set using an environment variable, settings file, and command-line argument.
 
-For more information, see [Use multiple environments](xref:fundamentals/environments).
+For more information, see <xref:fundamentals/environments>.
+
+## Hosting
+
+ASP.NET Core apps configure and launch a *host*, which is responsible for app startup and lifetime management.
+
+For more information, see <xref:fundamentals/host/index>.
+
+## Servers
+
+The ASP.NET Core hosting model doesn't directly listen for requests. The hosting model relies on an HTTP server implementation to forward the request to the app. The forwarded request is wrapped as a set of feature objects that can be accessed through interfaces. ASP.NET Core includes a managed, cross-platform web server, called [Kestrel](xref:fundamentals/servers/kestrel). Kestrel is commonly run behind a production web server, such as [IIS](https://www.iis.net/) or [Nginx](http://nginx.org) in a reverse proxy configuration. Kestrel can also be run as an edge server exposed directly to the Internet.
```

Comment:
I believe this last sentence is false for 2.0 and below. Or, at least, it wasn't recommended until the 2.1 release came along.

Reply:
Yes! I'll fix it.